docs/AboutUs.adoc

```diff
@@ -4,53 +4,34 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
-We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
+AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team.
+ 
```

Comment:
This - trailing whitespace - is causing the **failed - Travis build**

![capture](https://user-images.githubusercontent.com/12619937/52983470-40148900-3426-11e9-8d20-193cce633426.PNG)

Next time before you create a pull request, ensure that the Travis Build is successful.
I will accept this pull request for now.